api/types/saml.go

```diff
@@ -459,3 +475,11 @@ type SigningKeyPair struct {
 	// Cert is certificate in OpenSSH authorized keys format
 	Cert string `json:"cert"`
 }
+
+// EncryptionKeyPair is a key pair used for decryption SAML assertions.
+type EncryptionKeyPair struct {
```

Comment:
Rename this to `AsymmetricKeyPair` or `KeyPair` and use it instead of both `SingingKeyPair` and `EncryptionKeyPair`

Reply:
adjusted